### DISCUSSION TITLE
Custom validation implementation opinions

### DISCUSSION THREAD

**Discussion by martinjlowm** [colinhacks/zod#1779] [General]
Hi,

I just happened to look into some field validation and I was wondering why there's no validation implementation for phone numbers seeing that there are implementations for other things like uuid, email, url etc.? Any particular reason for why that is?

**Comment by JacobWeisenburger** [comment]
As far as I know, the rules for phone numbers are far more complex than emails or uuids. But you are always welcome to submit a PR.
